tests/unit/client/test_mixin_client.py

```diff
@@ -213,6 +217,58 @@ def test_dataframes(project_client, test_dataframe):
     assert dataframe_b.id in [d.id for d in dataframes]
 
 
+def test_dataframes_by_name(project_client, test_dataframe):
```

Comment:
Maybe logging one more dataframe with a different name would be good to add to this test to make sure it's grabbing only the ones with the name provided.